Fix + Test for  issue 22699

bouraqadi opened the pull request:
KeyCombination from an uppercase character  = lowercase character  + SHIFT

tesonep committed:
Fixing the lookup of extension packages that are not named exactly as the package.

demarey committed:
Merge 847e594c905d7674c50c8696ea3cec3c87753c1b

demarey committed:
Just fixing names from https://github.com/pharo-project/pharo/pull/1685/commits/847e594c905d7674c50c8696ea3cec3c87753c1b

demarey committed:
fix protocol name and call super tearDown at the last step

bouraqadi committed:
Fix + Test for  issue 22699
KeyCombination from an uppercase character  = lowercase character  + SHIFT

philippeback committed:
Fixed menubar consitency

* no more moving labels
* use the world menu highlight color for the menu
* add spaces for lining up labels

philippeback committed:
Set proper size for Pharo icon

pavel-krivanek committed:
- remove display scale factor for white tree forms because they are generated from already scaled forms

jecisc committed:
Add info about non default ssh port for gitlab.

Fixes https://pharo.fogbugz.com/f/cases/22728/Add-enough-informations-to-Metacello-to-load-a-project-from-a-gitlab-deployed-with-non-default-ssh-port

estebanlm committed:
Merge pull request #2037 from philippeback/22726-Pharo-icon-in-menus-is-larger-than-other-icons-20-pix-instead-of-16-making-its-row-height-different

22726 pharo icon in menus is larger than other icons 20 pix instead of 16 making its row height different

estebanlm committed:
fix for notification.

estebanlm committed:
Merge pull request #2043 from estebanlm/22731-notification-mail-has-an-error-on-version

22731-notification-mail-has-an-error-on-version

jecisc committed:
Clean Menubar

- Remove commented code
- Use right methods from the theme
- Remove useless method

Fixes https://pharo.fogbugz.com/f/cases/22733/Some-menubar-cleaning

jecisc committed:
Work on migration form pharo 6 to 7 + non regression tests

Fixes https://pharo.fogbugz.com/f/cases/22735/ZipArchive-class-isZipArchive-is-broken-in-P7

pavel-krivanek committed:
remove PharoWelcomePage and add menu registration to the WelcomeHelp

jecisc committed:
Remove option "window color..." that is half working.

Fixes https://pharo.fogbugz.com/f/cases/22738/Remove-Window-color-option

estebanlm committed:
Merge pull request #1741 from demarey/22313-Packages-are-created-for-extension-methods-that-does-not-are-exactly-the-package-name

22313-Packages-are-created-for-extension-methods-that-does-not-are-exactly-the-package-name

astares committed:
change BaselineOfMorphic

estebanlm committed:
Merge pull request #2044 from jecisc/22733-Some-menubar-cleaning

22733-Some-menubar-cleaning

astares committed:
switch from System-Settings to System-Settings-Core in references

astares committed:
split into System-Settings-Core and System-Settings-Browser again

astares committed:
tag uncategorized classes

astares committed:
22741 Cleanups for SystemVersionTest (lint)

use assert:equals:

https://pharo.fogbugz.com/f/cases/22741/

astares committed:
22742 Package ManifestHiedra should be tagged

tagging ManifestHiedra, no other code change - safe to integrate

https://pharo.fogbugz.com/f/cases/22742/

astares committed:
22744 Package manifest ManifestFuel should be tagged and commented

- tag the class and give a comment

https://pharo.fogbugz.com/f/cases/22744/

astares committed:
22745 ManifestHermes and HEBinaryReader should be tagged

https://pharo.fogbugz.com/f/cases/22745

MarcusDenker committed:
Merge pull request #2053 from astares/22745-ManifestHermes-and-HEBinaryReader-should-be-tagged

22745 ManifestHermes and HEBinaryReader should be tagged

jecisc committed:
Take Sven advice into account.

This also remove a Reference to File where FileReference should be used.

estebanlm committed:
Merge pull request #2049 from astares/22740-System-Settings-should-be-System-Settings-Core

22740 system settings should be system settings core

estebanlm committed:
Merge pull request #2048 from jecisc/22738-Remove-Window-color-option

22738-Remove-Window-color-option

astares committed:
trigger CI

estebanlm committed:
Merge pull request #2046 from jecisc/22735-ZipArchive-classisZipArchive-is-broken-in-P7

22735-ZipArchive-classisZipArchive-is-broken-in-P7

svenvc committed:
Add GTPlayground class>>#openUrl:

https://pharo.fogbugz.com/f/cases/22750/Add-GTPlayground-class-openUrl

MarcusDenker committed:
Merge pull request #2050 from astares/22741-Cleanups-for-SystemVersionTest-lint

22741 Cleanups for SystemVersionTest (lint)

MarcusDenker committed:
Improve RFLiteralVariableNode printing
https://pharo.fogbugz.com/f/cases/22751/Improve-RFLiteralVariableNode-printing

MarcusDenker committed:
Block execution: #arguments reification
  https://pharo.fogbugz.com/f/cases/22703/Block-execution-arguments-reification

estebanlm commented:
Hi @bouraqadi : the broken tests *are* related with this change. 
Can you please verify?

estebanlm committed:
Merge pull request #2047 from pavel-krivanek/22736-Clean-PharoWelcomePage

22736-Clean-PharoWelcomePage

MarcusDenker committed:
Merge pull request #2059 from MarcusDenker/22703-Block-execution-arguments-reification

22703-Block-execution-arguments-reification

MarcusDenker committed:
Merge pull request #2058 from MarcusDenker/22751-Improve-RFLiteralVariableNode-printing

22751-Improve-RFLiteralVariableNode-printing

MarcusDenker committed:
cleanup preamble generation + test for method Sequence
https://pharo.fogbugz.com/f/cases/22752/cleanup-preamble-generation-test-for-method-Sequence

MarcusDenker committed:
Merge pull request #2051 from astares/22742-Package-ManifestHiedra-should-be-tagged

22742 Package ManifestHiedra should be tagged

MarcusDenker committed:
RBParseErrorNode should pretty print thttps://pharo.fogbugz.com/f/cases/22753/RBParseErrorNode-should-pretty-print-the-faulty-text-if-it-has-ithe faulty text (if it has it)

MarcusDenker committed:
Merge pull request #2060 from MarcusDenker/22752-cleanup-preamble-generation--test-for-method-Sequence

22752-cleanup-preamble-generation--test-for-method-Sequence

guillep committed:
Use FileSystem file references instead of old FileStream

MarcusDenker committed:
Merge pull request #2057 from svenvc/22750-Add-GTPlayground-classopenUrl

22750-Add-GTPlayground-classopenUrl

MarcusDenker committed:
22755  Reflectivity: cleanups (tests, unify variable Naming)
https://pharo.fogbugz.com/f/cases/22755/Reflectivity-cleanups-tests-unify-variable-Naming

-  unify varName usage
- fix tests to use assert:equals:

MarcusDenker committed:
Merge pull request #2063 from MarcusDenker/22755-Reflectivity-cleanups-tests-unify-variable-Naming

22755-Reflectivity-cleanups-tests-unify-variable-Naming

MarcusDenker committed:
Merge pull request #2061 from MarcusDenker/22753-RBParseErrorNode-should-pretty-print-the-faulty-text-if-it-has-it

22753-RBParseErrorNode-should-pretty-print-the-faulty-text-if-it-has-it

astares committed:
22762 Cleanup TextEditorTest

- reduce double code by using setUp/tearDown
- tag and comment the class

https://pharo.fogbugz.com/f/cases/22762

astares committed:
22763 Tag classes in Text-Diff package

https://pharo.fogbugz.com/f/cases/edit/22763

astares committed:
22764 Cleanup EmbeddedFreeType-Tests and EmbeddedFreeType

- tagging classes
- implement isAbstract

https://pharo.fogbugz.com/f/cases/22764/

astares committed:
22765 Categorize uncategorized in Athens-Cairo

https://pharo.fogbugz.com/f/cases/22765

pavel-krivanek committed:
unsubscribe Transcript window during its closing

astares committed:
22767 Categorize uncategorized classes in Tool-Finder

only tagging - no change in behavior. safe to integrate

https://pharo.fogbugz.com/f/cases/22767

MarcusDenker committed:
Merge pull request #2067 from astares/22763-Tag-classes-in-Text-Diff-package

22763 Tag classes in Text-Diff package

MarcusDenker committed:
Merge pull request #2066 from astares/22762-Cleanup-TextEditorTest

22762 Cleanup TextEditorTest

astares committed:
22768 Categorize classes in OSWindow-SDL2-Base

tagging only - no change in behavior, safe to integrate

https://pharo.fogbugz.com/f/cases/22768/

bouraqadi commented:
Looking at it...

MarcusDenker committed:
Merge pull request #2071 from pavel-krivanek/22766-Transcript-window-is-leaking

22766-Transcript-window-is-leaking

MarcusDenker committed:
Merge pull request #2070 from astares/22767-Categorize-uncategorized-classes-in-Tool-Finder

22767 Categorize uncategorized classes in Tool-Finder

pavel-krivanek committed:
fix unixVirtualKey table

MarcusDenker committed:
Merge pull request #2041 from jecisc/22728-Add-enough-informations-to-Metacello-to-load-a-project-from-a-gitlab-deployed-with-non-default-ssh-port

22728-Add-enough-informations-to-Metacello-to-load-a-project-from-a-gitlab-deployed-with-non-default-ssh-port

MarcusDenker committed:
Merge pull request #2069 from astares/22765-Categorize-uncategorized-in-

22765 Categorize uncategorized in Athens-Cairo

MarcusDenker committed:
Merge pull request #2074 from pavel-krivanek/22769-KeyboardKey-classunixVirtualKeyTable-uses-KeyTable-instead-of-UnixKeyTable-

22769-KeyboardKey-classunixVirtualKeyTable-uses-KeyTable-instead-of-UnixKeyTable-

MarcusDenker committed:
Merge pull request #2068 from astares/22764-Cleanup-EmbeddedFreeType-Tests-and-EmbeddedFreeType

22764 Cleanup EmbeddedFreeType-Tests and EmbeddedFreeType

astares committed:
fix uncategorized

MarcusDenker committed:
Merge pull request #2062 from guillep/22754-Exception-while-opening-a-MCSubDirectoryRepository

22754-Exception-while-opening-a-MCSubDirectoryRepository

MarcusDenker committed:
Merge pull request #2072 from astares/22768-Categorize-classes-in-OSWindow-SDL2-Base

22768 Categorize classes in OSWindow-SDL2-Base

MarcusDenker committed:
Merge pull request #2052 from astares/22744-Package-manifest-ManifestFuel-should-be-tagged-and-commented

22744 Package manifest ManifestFuel should be tagged and commented

guillep committed:
Fix issue 22772

MarcusDenker committed:
Merge pull request #2075 from guillep/22772-chooseExistingFileReferenceextensionspathpreview-returning-wrong-FileReference

22772-chooseExistingFileReferenceextensionspathpreview-returning-wrong-FileReference

astares committed:
22778 Tag Monticello-OldDataStreamCompatibility in Manifest

tag
  - manifest class into "Manifest" tag
 - the rest into "Base" as it is base functionality of the package

https://pharo.fogbugz.com/f/cases/22778

astares committed:
22780 Fix uncategorized classes in MonticelloGUI

only recategorization. no change in behavior - safe to merge

https://pharo.fogbugz.com/f/cases/22780

Ducasse committed:
fixing comment on yourself

Ducasse committed:
Merge pull request #2038 from pavel-krivanek/22723-Ugly-triangles-in-trees-with-medium-fontsUntitled

22723-Ugly-triangles-in-trees-with-medium-fontsUntitled

Ducasse committed:
first cleaning and adding of executable examples.

Ducasse committed:
Doing a pass on comments!

Ducasse committed:
Merge pull request #2079 from Ducasse/22756-Yourself-comment-is-wrong-

22756-Yourself-comment-is-wrong-

MarcusDenker committed:
Merge pull request #2077 from astares/22778-Tag-Monticello-OldDataStreamCompatibility-in-Manifest-

22778 Tag Monticello-OldDataStreamCompatibility in Manifest

astares committed:
trigger ci

astares committed:
22786 - Unused result in testExecutionOfWrongFFIMethodShouldRaiseAnError

- remove result variable in testExecutionOfWrongFFIMethodShouldRaiseAnError
- add comment to ffiCopyStringWithError: to:

https://pharo.fogbugz.com/f/cases/22786/

Ducasse committed:
Fixing size

Ducasse committed:
Merge pull request #2080 from Ducasse/22782-Bag-comments-deserve-executable-comments

22782-Bag-comments-deserve-executable-comments

MarcusDenker committed:
Merge pull request #2081 from astares/22786-Unused-result-in-testExecutionOfWrongFFIMethodShouldRaiseAnError

22786 - Unused result in testExecutionOfWrongFFIMethodShouldRaiseAnError

astares committed:
22760 RBRefactoryClassChange should define #isAbstract with true

- implement #isAbstract
- adjust comment

https://pharo.fogbugz.com/f/cases/22760/

astares committed:
22790 Remove unnecessary spaces in OpenToolTest

- unnecessary spaces removed, no change in behavior

https://pharo.fogbugz.com/f/cases/22790/Remove-unnecessary-spaces-in-OpenToolTest

astares committed:
22791 Cleanup TextDiffBuilderTest

- use assert:equals:

https://pharo.fogbugz.com/f/cases/22791/Cleanup-TextDiffBuilderTest

astares committed:
22792 Cleanup MessageTallyTest

- simple cleanups like use assert:equals

https://pharo.fogbugz.com/f/cases/22792/Cleanup-MessageTallyTest

astares committed:
22793 Cleanup AndreasSystemProfilerTest

- use assert:equals:

https://pharo.fogbugz.com/f/cases/22793/Cleanup-AndreasSystemProfilerTest

astares committed:
22794 Cleanup testExtractAllKinds

- use assert:equals:

https://pharo.fogbugz.com/f/cases/22794/Cleanup-testExtractAllKinds

astares committed:
22795 Lint cleanup in Bitmap

- category change to satisfy lint and follow object protocol

astares committed:
22796 Fix testResettingTheSelection

- do not use shouldnt:raise: in  test method #testResettingTheSelection

https://pharo.fogbugz.com/f/cases/22796/Fix-testResettingTheSelection

MarcusDenker committed:
Merge pull request #2089 from astares/22794-Cleanup-testExtractAllKinds

22794 Cleanup testExtractAllKinds

MarcusDenker committed:
Merge pull request #2085 from astares/22790-Remove-unnecessary-spaces-in-OpenToolTest

22790 Remove unnecessary spaces in OpenToolTest

MarcusDenker committed:
Merge pull request #2090 from astares/22795-Lint-cleanup-in-Bitmap

22795 Lint cleanup in Bitmap

MarcusDenker committed:
Merge pull request #2086 from astares/22791-Cleanup-TextDiffBuilderTest

22791 Cleanup TextDiffBuilderTest

MarcusDenker committed:
Merge pull request #2087 from astares/22792-Cleanup-MessageTallyTest

22792 Cleanup MessageTallyTest

MarcusDenker committed:
Merge pull request #2091 from astares/22796-Fix-testResettingTheSelection

22796 Fix testResettingTheSelection

MarcusDenker committed:
Merge pull request #2088 from astares/22793-Cleanup-AndreasSystemProfilerTest

22793 Cleanup AndreasSystemProfilerTest

astares committed:
22801 Small cleanups in TRBProgramNodeVisitor

- use chaining

https://pharo.fogbugz.com/f/cases/22801/Small-cleanups-in-TRBProgramNodeVisitor

MarcusDenker committed:
Merge pull request #2093 from astares/22801-Small-cleanups-in-TRBProgramNodeVisitor

22801 Small cleanups in TRBProgramNodeVisitor

MarcusDenker committed:
Merge pull request #2084 from astares/22760-RBRefactoryClassChange-should-define-isAbstract-with-true

22760 RBRefactoryClassChange should define #isAbstract with true

MarcusDenker committed:
Merge pull request #2078 from astares/22780-Fix-uncategorized-classes-in-MonticelloGUI

22780 Fix uncategorized classes in MonticelloGUI

bouraqadi committed:
Merge f50d988b3428cd335b5308e566196f33a571315c

bouraqadi committed:
Integrated issue 22699 with the latest Pharo 7 and fixing collisions with Glamour and CharacterKeyCombinationTest

bouraqadi commented:
Did updated the branch + more fixes for the integration to avoid breaking other tests.
Thanks Pablo for the help :-)
